src/script/components/manifest-options.ts

```diff
@@ -436,7 +477,14 @@ export class AppManifest extends LitElement {
   }
 
   openUploadModal() {
-    console.log('open upload modal', event);
+    this.uploadModalOpen = true;
+    this.requestUpdate();
```

Comment:
Now, this one is interesting, not sure why we need requestUpdate here, but im guessing if you didnt have this it probably wasnt detecting the change to the uploadModalOpen prop?

Reply:
hmm, I don't remember, I think I added it because some wonkiness is reloading tbh.

Reply:
Got ya, I'd say let's just double check if it can be removed, otherwise I think it's fine.